packages/sdks/snippets/gen1-remix/app/routes/editable-region.tsx

```diff
@@ -0,0 +1,49 @@
+// routes/advanced-child.tsx
+import { Builder, BuilderComponent, builder } from "@builder.io/react";
+import type { LoaderFunctionArgs } from "@remix-run/node";
+import { useLoaderData } from "@remix-run/react";
+import { CustomColumns } from "./components/CustomColumns";
+
+builder.init("ee9f13b4981e489a9a1209887695ef2b");
+
+Builder.registerComponent(CustomColumns, {
+  name: "MyColumns",
+  inputs: [
+    {
+      name: "column1",
+      type: "uiBlocks",
+      broadcast: true,
+      hideFromUI: true,
+      defaultValue: {
+        blocks: [],
+      },
+    },
+    {
+      name: "column2",
+      type: "uiBlocks",
+      broadcast: true,
+      hideFromUI: true,
+      defaultValue: {
+        blocks: [],
+      },
+    },
+  ],
+});
+
+export const loader = async ({ request }: LoaderFunctionArgs) => {
+  const page = await builder
+    .get("editable-regions", {
+      userAttributes: {
+        urlPath: `/${request.url.split("/").pop()}`,
```

Comment:
instead of `request.url.split("/").pop()`, we can rely on remix's file naming conventions to extract the URL path as a param. See https://github.com/BuilderIO/builder/blob/b346d8733b5b606e607c6ed5b27a2d649af08e24/packages/sdks/e2e/remix/app/routes/($slug).tsx#L44-L53

by naming the file `($slug).tsx`, remix will automatically pass the URL path value to `params.slug`. This is far cleaner as an approach, so let's do things this way for Remix moving forward

Reply:
We do it in the snippets already doo. See: https://github.com/BuilderIO/builder/blob/b346d8733b5b606e607c6ed5b27a2d649af08e24/packages/sdks/snippets/gen1-remix/app/routes/announcements.$slug.tsx#L12-L21

Reply:
i just realized this suggestion of mine doesn't work because the routes you added in this PR are hardcoded with exact URLs 🤔

forget what i said for now, and let's stick with your approach